tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -3292,28 +3292,17 @@ class HLSLExternalSource : public ExternalSemaSource {
           break;
         }
       } else if (kind == AR_OBJECT_RAY_QUERY) {
-        ClassTemplateDecl* typeDecl = nullptr;
-        AddRayQueryTemplate(*m_context, &typeDecl, &recordDecl);
-        DXASSERT(typeDecl != nullptr, "AddRayQueryTemplate failed to return the object declaration");
-        typeDecl->setImplicit(true);
```

Comment:
Those `setImplicit` are now done by the builder. It makes sense because "implicit" means that it has no source code location, which applies to all builtin types.